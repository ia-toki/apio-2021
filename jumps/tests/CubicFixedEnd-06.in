209636825ecd2c38299498cd26890ef71dd128c2
200 200
198 197 196 192 191 190 186 185 184 180 179 178 174 173 172 168 167 166 162 161 160 156 155 154 150 149 148 144 143 142 138 137 136 132 131 130 126 125 124 120 119 118 114 113 112 108 107 106 102 101 100 96 95 94 90 89 88 84 83 82 78 77 76 72 71 70 66 65 64 60 59 58 54 53 52 48 47 46 42 41 40 36 35 34 30 29 28 24 23 22 18 17 16 12 11 10 6 5 4 1 2 3 7 8 9 13 14 15 19 20 21 25 26 27 31 32 33 37 38 39 43 44 45 49 50 51 55 56 57 61 62 63 67 68 69 73 74 75 79 80 81 85 86 87 91 92 93 97 98 99 103 104 105 109 110 111 115 116 117 121 122 123 127 128 129 133 134 135 139 140 141 145 146 147 151 152 153 157 158 159 163 164 165 169 170 171 175 176 177 181 182 183 187 188 189 193 194 195 199 200
99 99 199 199
14 107 130 130
58 101 190 190
9 56 134 134
115 184 189 189
6 31 65 65
138 167 184 184
3 28 91 91
70 182 185 185
55 100 190 190
47 113 190 190
90 194 198 198
23 30 75 75
177 188 195 195
46 103 136 136
9 48 53 53
68 129 136 136
46 131 133 133
0 8 190 190
12 18 194 194
54 102 109 109
28 60 69 69
22 35 141 141
2 26 184 184
31 112 140 140
10 12 26 26
31 64 134 134
27 58 97 97
21 143 184 184
153 198 199 199
20 68 163 163
32 165 166 166
8 66 103 103
58 91 188 188
11 111 138 138
3 108 197 197
30 132 197 197
76 179 192 192
56 56 116 116
1 5 51 51
57 157 160 160
73 182 192 192
167 168 191 191
28 153 169 169
82 133 187 187
82 185 191 191
0 0 108 108
8 183 184 184
29 160 191 191
40 121 195 195
74 151 173 173
49 172 185 185
35 131 137 137
134 135 184 184
24 52 84 84
132 177 181 181
101 156 177 177
52 91 169 169
106 113 123 123
67 70 122 122
40 140 185 185
8 13 198 198
30 182 198 198
3 41 185 185
137 166 176 176
37 77 168 168
12 14 198 198
122 175 190 190
68 109 177 177
145 154 168 168
0 29 191 191
51 62 185 185
59 118 125 125
44 49 112 112
28 89 106 106
8 198 199 199
61 67 194 194
40 136 193 193
0 45 106 106
39 114 180 180
37 194 198 198
0 1 8 8
27 70 194 194
22 147 158 158
143 172 197 197
136 175 197 197
1 105 198 198
134 170 193 193
54 169 188 188
22 23 172 172
29 36 154 154
99 128 191 191
83 94 187 187
5 19 66 66
48 78 90 90
128 160 161 161
136 147 198 198
44 196 197 197
72 80 191 191
12 62 140 140
14 74 75 75
23 50 61 61
71 134 158 158
22 103 140 140
13 39 124 124
64 107 127 127
9 64 98 98
28 149 171 171
1 3 65 65
53 63 156 156
72 158 175 175
25 179 188 188
24 38 125 125
41 188 191 191
20 158 162 162
124 176 187 187
92 107 138 138
15 55 56 56
8 21 128 128
29 32 61 61
108 117 175 175
21 57 132 132
6 84 157 157
65 90 129 129
95 105 138 138
61 129 134 134
48 125 187 187
46 51 156 156
2 94 178 178
46 141 161 161
10 72 117 117
39 97 167 167
82 82 187 187
49 146 179 179
58 132 177 177
23 119 171 171
6 183 193 193
24 81 174 174
148 176 185 185
4 148 171 171
35 105 199 199
10 23 155 155
26 120 148 148
105 160 163 163
51 101 120 120
33 160 179 179
48 67 134 134
6 109 173 173
10 25 172 172
45 135 178 178
106 135 136 136
75 89 120 120
38 107 173 173
22 79 83 83
108 130 141 141
84 181 186 186
165 191 195 195
76 136 150 150
7 22 31 31
6 83 165 165
108 115 185 185
97 190 199 199
31 37 111 111
93 162 186 186
50 136 177 177
51 75 194 194
19 101 131 131
10 15 118 118
26 169 173 173
24 96 142 142
93 114 196 196
120 135 160 160
95 193 197 197
22 88 126 126
161 186 188 188
57 95 130 130
91 172 177 177
31 128 158 158
6 21 101 101
12 55 196 196
19 28 93 93
139 178 192 192
5 6 34 34
11 27 179 179
57 127 137 137
10 53 192 192
33 55 132 132
12 30 108 108
81 88 147 147
54 194 197 197
16 47 88 88
62 76 172 172
5 109 114 114
32 36 45 45
79 131 193 193
16 165 199 199
4 14 35 35
81 144 164 164
23 82 180 180
90 95 190 190
