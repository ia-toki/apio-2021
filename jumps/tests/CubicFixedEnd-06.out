fc5005cd102fffc504bc1615010ebf88823d65f3
OK
67
2
34
-1
4
-1
12
-1
3
32
26
3
-1
5
2
-1
4
2
-1
6
2
-1
-1
2
3
-1
-1
-1
5
1
2
1
-1
33
1
3
20
9
-1
-1
3
7
16
2
37
5
-1
1
15
25
16
10
3
33
-1
3
14
16
7
-1
19
5
11
3
8
5
8
11
31
10
3
26
3
-1
-1
1
39
24
-1
14
3
-1
16
3
18
16
1
16
14
-1
-1
43
48
-1
-1
1
34
1
44
3
-1
-1
17
3
-1
2
-1
1
-1
7
12
7
-1
3
1
8
21
-1
-1
-1
39
-1
2
1
22
3
25
3
2
7
-1
6
48
21
25
1
2
1
7
1
24
-1
2
3
1
10
3
3
-1
17
1
1
9
-1
8
4
3
10
-1
1
48
7
-1
16
19
32
3
-1
3
2
55
17
4
1
2
2
4
3
-1
7
-1
10
-1
3
3
3
-1
-1
20
3
-1
24
1
-1
42
12
-1
14
3
55
